package.json

```diff
@@ -20,6 +20,7 @@
     "lint": "eslint gulpfile.js src/*.js src/**/*.js",
     "build": "gulp",
     "docs": "node scripts/build-docs.js",
+    "examples": "node scripts/build-examples.js",
```

Comment:
Do we need a separate build script for the examples?  Might just be part of `start` `start:watch`.  I don't have a strong opinion here.  Just raising it as a thought.

Reply:
Yeah... reflecting on it, building the examples without first running the start script wouldn't make much sense.

Reply:
I think we should remove this script then since you removed the cli support from the script.

Reply:
Oh, that's strange I could've sworn I removed it with my most recent commit. I'll fix it up, good catch!